components/core/src/clp_s/ArchiveReader.cpp

```diff
@@ -4,20 +4,30 @@
 #include <string_view>
 
 #include "archive_constants.hpp"
+#include "InputConfig.hpp"
 #include "ReaderUtils.hpp"
 
 using std::string_view;
 
 namespace clp_s {
-void ArchiveReader::open(string_view archives_dir, string_view archive_id) {
+void ArchiveReader::open(Path const& archive_path, NetworkAuthOption const& network_auth) {
```

Comment:
It gets used in the follow-up PR when we start supporting single file archive reading over the network.

I think it makes sense to keep it because Path requires NetworkAuth in the general case, and it makes the diff for the next PR a bit smaller.